src/toast/toaster.js

```diff
@@ -41,6 +41,15 @@ export class ToasterContainer extends React.Component<
 
   constructor(props: ToasterPropsT) {
     super(props);
+
+    if (__DEV__) {
+      if (toasterRef !== undefined) {
+        // eslint-disable-next-line no-console
+        console.warn(
+          'A ToasterContainer already exists in your application. It is not recommended to have more than one ToasterContainer in an application.',
```

Comment:
I have a bit of a problem with this one. We will be spammed with that message in development, as we plan to have multiple toasters in the near future. I know it's not recommended, but there are valid uses cases and technically there should be no issue if done with a `ref`. Maybe the check could be more sophisticated and check if each `ToasterContainer` has a `ref` before creating the warning.

See a working sandbox here: https://codesandbox.io/s/toaster-basic-usage-4rqld

Also discussed in #1769

Reply:
good point, I didn't consider that it could be used that way. I took it out since there isn't an easy way to check for "ref" prop (could use React.forwardRef but... that will be different issue)

Also just thought if placement could be controlled by the toast itself (the ToasterContainer prop can jsut be default like autoHideTimeout is) that would cover your use case with a single ToasterContainer as well. It shouldn't be too hard to sort toasts based on placement and render them separately. But again, another time, another issue :)

Reply:
> Also just thought if placement could be controlled by the toast itself (the ToasterContainer prop can jsut be default like autoHideTimeout is) that would cover your use case with a single ToasterContainer as well. It shouldn't be too hard to sort toasts based on placement and render them separately. But again, another time, another issue :)

I'd personally love to see that (in a different PR of course). It would make it simpler and then this warning could be reintroduced and the old behavior deprecated in the warning with a link to the documentation for that new feature. But as you may read in the issue I created to ask for this, it did not seem like this was a feature the Uber team wanted as it was not good practice, but they may be more open to it now (as they had a case where they helped someone implement it after I did my question)?